styles/components/_form.scss

```diff
@@ -55,8 +63,8 @@ textarea,
   line-height: 40px;
 }
 
-.medium-editor-element {
-  padding-top: $padSm;
+.trumbowyg div[contenteditable].rows6 {
+  height: 18ex;
```

Comment:
I don't recommend using the x-height for this, as some fonts may have a total height larger than twice the x-height, which will throw things off. Using em or rem as the unit would be better.